arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -121,8 +121,8 @@
         "tags": [
           "Registries"
         ],
-        "description": "Creates a container registry with the specified parameters.",
-        "operationId": "Registries_Create",
+        "description": "Creates or updates a container registry with the specified parameters.",
+        "operationId": "Registries_CreateOrUpdate",
```

Comment:
If you're allowing updating of any field other than the tags, break this into ```Create``` and ```Update``` (which you already seem to be doing)